### PR TITLE
feature: Stripe integration

### PR DESCRIPTION
**Pull request by IceToast**
This PR implements the stripe integration as mentioned in this issue #259.

Stripe Checkout would look like this, where as the Credit Product is one billing line and the calcuated taxes in cp.gg the second line. 
The Tax feature of stripe will not be used as it does need much more handling with stripe and is unnecessary. It would be useful the whole tax logic is getting handled by the payment providers (paypal, stripe etc.) and not cp.gg.

Screenshot:
unknown

### CODE REVIEW

**Comment by 1day2die** [comment]
add "stripe integration" to the readme.md :P

**Comment by IceToast** [comment]
This is pretty much done 💯

File: resources/views/store/checkout.blade.php [hunk 1]
```diff
@@ -71,20 +75,22 @@
...
                                 <table class="table table-striped">
                                     <thead>
-                                    <tr>
-                                        <th>{{__('Quantity')}}</th>
-                                        <th>{{__('Product')}}</th>
-                                        <th>{{__('Description')}}</th>
-                                        <th>{{__('Subtotal')}}</th>
-                                    </tr>
+                                        <tr>
+                                            <th>{{ _('Quantity') }}</th>
```
Comment (1day2die) [inline resources/views/store/checkout.blade.php:79]: you missed a second underscore on all the translations here.

File: resources/views/store/checkout.blade.php [hunk 2]
```diff
@@ -34,33 +37,34 @@
...
+                                    <strong>{{ Auth::user()->name }}</strong><br>
+                                    {{ _('Email') }}: {{ Auth::user()->email }}
                                 </address>
                             </div>
                             <!-- /.col -->
                             <div class="col-sm-4 invoice-col">
-                                <b>{{__('Status')}}</b><br>
-                                <span class="badge badge-warning">{{__('Pending')}}</span><br>
-{{--                                <b>Order ID:</b> 4F3S8J<br>--}}
+                                <b>{{ _('Status') }}</b><br>
```
Comment (1day2die) [inline resources/views/store/checkout.blade.php:65]: you missed a second underscore on all the translations here.

File: resources/views/store/checkout.blade.php [hunk 3]
```diff
@@ -34,33 +37,34 @@
...
+                                <span class="badge badge-warning">{{ _('Pending') }}</span><br>
```
Comment (1day2die) [inline resources/views/store/checkout.blade.php:66]: you missed a second underscore on all the translations here.

File: resources/views/store/checkout.blade.php [hunk 4]
```diff
@@ -71,20 +75,22 @@
...
-                                        <th>{{__('Quantity')}}</th>
-                                        <th>{{__('Product')}}</th>
-                                        <th>{{__('Description')}}</th>
-                                        <th>{{__('Subtotal')}}</th>
-                                    </tr>
+                                        <tr>
+                                            <th>{{ _('Quantity') }}</th>
+                                            <th>{{ _('Product') }}</th>
+                                            <th>{{ _('Description') }}</th>
+                                            <th>{{ _('Subtotal') }}</th>
```
Comment (1day2die) [inline resources/views/store/checkout.blade.php:82]: till here

File: resources/views/store/checkout.blade.php [hunk 5]
```diff
@@ -95,35 +101,50 @@
...
+                                        <img class="mb-3" height="50"
+                                            src="{{ url('/images/paypal_logo.png') }}"></br>
 
-                                <img src="https://www.paypalobjects.com/digitalassets/c/website/logo/full-text/pp_fc_hl.svg" alt="Paypal">
+                                        <input x-model="paymentMethod" type="radio" id="paypal" value="paypal"
+                                            name="payment_method">
+                                        </input>
+                                    </label>
+
+                                    <label class="ml-5 text-center " for="stripe">
```
Comment (1day2die) [inline resources/views/store/checkout.blade.php:116]: What if stripe isnt configured?

**Pull request merged**